python/cugraph/cugraph/gnn/graph_store.py

```diff
@@ -16,8 +16,9 @@
 import cugraph
 from cugraph.experimental import PropertyGraph
 from cugraph.community.egonet import batched_ego_graphs
-from cugraph.utilities.utils import sample_groups
+from cugraph.sampling import uniform_neighbor_sample
 import cupy as cp
+from functools import cached_property
```

Comment:
This is nice, I forgot it's in the std library. I will start using this elsewhere.